AdvLoggerPkg/Test/Mock/Library/GoogleTest/MockAdvancedLoggerAccessLib/MockAdvancedLoggerAccessLib.inf

```diff
@@ -0,0 +1,34 @@
+## @file
+# Google Test mocks for AdvancedLoggerAccessLib
+#
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = MockAdvancedLoggerAccessLib
+  FILE_GUID                      = 476554ce-578a-5a9d-87ad-52d1fa226108
+  MODULE_TYPE                    = HOST_APPLICATION
+  VERSION_STRING                 = 1.0
+  LIBRARY_CLASS                  = AdvancedLoggerAccessLib
+
+#
+# The following information is for reference only and not required by the build tools.
+#
+#  VALID_ARCHITECTURES           = IA32 X64
+#
+
+[Sources]
+  MockAdvancedLoggerAccessLib.cpp
+
+[Packages]
+  AdvLoggerPkg/AdvLoggerPkg.dec
+  MdePkg/MdePkg.dec
+  UnitTestFrameworkPkg/UnitTestFrameworkPkg.dec
+
+[LibraryClasses]
+  GoogleTestLib
+
+[BuildOptions]
+  MSFT:*_*_*_CC_FLAGS = /EHs /bigobj

```

Comment:
Why are these needed? From the actual lib?

Reply:
They shouldn't be needed. Removed them